### PR TITLE
Add a bfcache time limit to privacy section.

### LINKED ISSUE #3: [Declarative Beacon API] Time limit on retaining beacons for bfcache'd pages
My understanding is that with the Declarative Beacon API, if a page is kept in the bfcache for an extended period of time, it's still eligible to send the beacon once the page is evicted from the bfcache, which might be a long time after the user has navigated away from the page. 

Would it be possible to instead send the beacon for bfcache'd pages after they've been in the bfcache for a minute or so? I think the page would want to check when waking from the bfcache to see if the beacon was already sent or not (by checking the `state` field of the beacon from the explainer) -- if it's already sent, the page may want to create a new beacon (at its choice).

### PR DESCRIPTION
**Pull request by caraitto**
Closes #3

### CODE REVIEW

**Comment by caraitto** [comment]
Hi, friendly ping?

**Comment by mingyc** [comment]
Sorry I thought that my comment was sent out but it was not.

File: explainers/beacon_api.md [hunk 1]
```diff
@@ -212,6 +212,7 @@ Specifically, beacons will have the following privacy requirements:
 *   Beacons registered in an incognito session do not persist to disk.
 *   Follow third-party cookie rules for beacons.
 *   Post-unload beacons are not sent if background sync is disabled for a site.
+*   If a page is suspended (for instance, as part of a [bfcache](https://web.dev/bfcache/)), beacons should be sent within 10 minutes or less of suspension, to keep the beacon send temporally close to the user's page visit.
```
[the remark this answers is not in the document]
Comment (caraitto) [reply explainers/beacon_api.md:215]: Fixed in other PR.

**Comment by caraitto** [comment]
My Github-fu is lacking -- I made a new pull request https://github.com/WICG/unload-beacon/pull/15 because I couldn't figure out how to handle the complex merge (due to the file move). Sorry for the inconvenience.

**Pull request closed without merging**